sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1165,7 +1170,7 @@ public void tearDown() throws Exception {
       if (bigtableWriter != null) {
         bigtableWriter.close();
         bigtableWriter = null;
-        FACTORY_INSTANCE.releaseWriteService(serviceEntry);
+        factory.releaseWriteService(serviceEntry);
```

Comment:
can you null out the entry as well? mismatched refcounts are harder to debug than NPEs